manifests/overlays/prod/applications/data_hub/dh-psi-monitoring.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   source:
     path: prometheus/overlays/prod/
     repoURL: https://github.com/AICoE/internal-data-hub.git
-    targetRevision: master
+    targetRevision: main
```

Comment:
any reason this one is `master` instead of `HEAD` like the rest of the app overlays?

Reply:
Fixed.